### DISCUSSION TITLE
Issue in compiling FLOWUnsteady

### DISCUSSION THREAD

**Discussion by aeroarun** [byuflowlab/FLOWUnsteady#52] [Q&A] [answered]
By following the installation instructions, I could successfully install FLOWExaFMM, FLOWVPM and other packages (AirfoilPrep.jl, FLOWVLM, BPM.jl, FLOWNoise). When I finally started installing the FLOWUnsteady package, I ended up with the following error. 

![FLOWUnsteadyInstallationError.PNG](https://user-images.githubusercontent.com/22278690/230670548-a73e9e16-a15c-44af-b8fc-8868ecc3c4bf.PNG)

Kindly help me in resolving this issue. Thank you.

**Comment by EdoAlvarezR** [comment, marked as answer]
Issue resolved in #53
